src/ethereum/wallet.rs

```diff
@@ -145,15 +145,16 @@ impl Wallet {
             .send_raw_transaction(transaction_hex)
             .await?;
 
+        #[allow(unreachable_patterns)]
```

Comment:
Please add a comment when adding `allow` attributes.
Also, the compiler is right, the last match branch is unreachable, just delete it.